docs/index.md

```diff
@@ -592,7 +592,32 @@ $ # The environment variables have been unset.
 
 #### inspect
 
-Inspect information about a machine.
+```
+Usage: docker-machine inspect [OPTIONS] [arg...]
+
+Inspect information about a machine
+
+Description:
+   Argument is a machine name. Will use the active machine if none is provided.
+
+Options:
+   --format, -f 	Format the output using the given go template.
+```
+
+By default, this will render all results in a JSON array. If a format is
```

Comment:
Yeah that was totally a copy-paste error - my bad ;)

I think I'll issue a PR _Real Soon Now_ for multi-machine support for `inspect`. Cuz while I was writing that I was thinking how awesome it would be. ;)